contest/templatetags/contest_extras.py

```diff
@@ -32,3 +38,72 @@ def has_auth(user,contest_id):
     return user_info.has_c_ownership(user,contest)
 
 register.filter("has_auth",has_auth)
+
+#check if user is judge or admin
+@register.filter
+def has_judge_auth(user):
```

Comment:
I think this function can be called in template file directly.(do not need to use filter)